5-unexpected-files/.dockerignore

```diff
@@ -0,0 +1,4 @@
+app/.env
+tests/
```

Comment:
Something to bear in mind on a delivery, if the image is being used to run tests as well as run the app, the tests files might be left in on a 'development' cut but the out file should always be excluded.
Production deployments definitely will be lighter and more secure without tests.